store/models/job_spec.go

```diff
@@ -208,25 +209,48 @@ type TaskSpec struct {
 	Params        JSON     `json:"-"`
 }
 
-// UnmarshalJSON parses the given input and updates the TaskSpec.
+//UnmarshalJSON parses the given input and updates the TaskSpec.
```

Comment:
Nitpick: I think the expected style is a single space after `//` (linter should pick this up)